textAdventure.js

```diff
@@ -0,0 +1,229 @@
+        console.log('\nYou run the troll through with your sword\n but the wounds close up quickly\n "HAHAHA," it says, "YOUR MORTAL WEAPONS WILL NOT WORK ON ME"\nit throws you back and resumes its previous stance')
+
+    }else if(choice==='pet bear' && currentRoom.description==='a sleeping bear.  It is very cute'){
+       console.log('the bear begins to stir.  It awakens and looks\ncuriously at you')
+       cave.description='The bear is awake, and looks\ncuriously at you.'
+    }else if(choice==='pet bear'&& currentRoom.description==='The bear is awake, and looks\ncuriously at you.'){
+        console.log("the bear's hind leg begins to shake uncontrollably.\nhe gives you a big slobbery kiss")
+        cave.description='a happy bear :)'
+    }else if(choice==='pet bear'&& currentRoom.name==cave.name&&currentRoom.description!=='The cave is very dark.  The growling is much louder'){
+        console.log("the bear licks your hand as you pet it")
+        cave.description='a happy bear :)'
+    }else if(choice==='read sign' && currentRoom.name===meadow2.name){
+        console.log('The sign reads:\n <----Bear Cave  Grandmas Pie Palace---->')
+    }else if (choice==='enter' || 'open door' &&currentRoom.name===grandmasHouse.name){
+        enterRoom('east');
+        if(currentRoom.name===winRoom.name){
+            console.log(`Thanks for playing, ${player.name}!`)
+            process.exit();
+        }
+    }else if (choice==='exit'){
+            process.exit()
+    }else console.log("\nSorry, I don't understand")
+    action();
+ }
+
+async function enterRoom(dir){ //dir = north,east,south,west
+        if(currentRoom[dir]){
+           currentRoom= nextRoom[currentRoom[dir]];
+           console.log("\n"+currentRoom.description + '\n');
+        } else console.log('\n Cannot go that way')
+}
+async function pickup(item){
+    if(currentRoom.items.includes(item)){
+        let i= currentRoom.items.indexOf(item);
+        console.log(`\nyou picked up ${currentRoom.items[i]}`);
+        player.inventory.push(hasItem[currentRoom.items[i]]);
+        currentRoom.items.splice(i,1);
+    } else console.log('\nitem unavailable')
+}
+async function drop(item){
+if (player.inventory.includes(item)){
+    let i= player.inventory.indexOf(item);
+    console.log(`\nyou dropped ${player.inventory[i].name}`);
+    currentRoom.items.push(player.inventory[i].name.toString())
+    player.inventory.splice(i,1);
+    }
+}
+async function examine(item){
+    if (player.inventory.includes(item)){
+        console.log(item.description)
+    } else console.log("\nYou don't have that item")
+}
+
+
+
+    /**tweak this to make more sense for the zorkington project
+     * make items have a description!  'inspect sword'
+     * --------->>>>items should be an array maybe?  or an object with more than one;
+     rooms need an inventory! ARRAY
+     Inventory, AND instructions
+     maybe roses at a win condition
```

Comment:
Ahhh, future features. A good idea to comment these in, but don't forget to remove the comments when you implement them, or you might accidentally implement them multiple times.